### ISSUE TITLE
Running de Bruijn graph code on Windows

### ISSUE THREAD

**Issue by mhassankhan90** [rchikhi/debruijn#2]
Hello Dear,
Greetings,

Can we use this code in Windows environment using Microsoft Visual Studio or MinGW compiler?

I have compiled and tested this code in Linux. It works well. Now, I am trying to run it on Windows. I have compiled it using MinGW compiler 8.1. It doesn't give any error during compilation except getopt argument not found which can be resolved by adding getopt.h file. 
When I run it in Windows on same data set that I used in Linux, it doesn't generate files with same size. The nodes and edges file generated in Windows contains only few entries where in Linux they are complete. I have also tried replacing the string cmd =" mv" with "move" which is supported in Windows cmd but it produces the same results. I am also attaching the last few lines of output it generates.
Kindly help me resolving the issue.

Thanks
![output](https://user-images.githubusercontent.com/60148534/76306090-1dcb0300-62e8-11ea-99ad-4df271ff1bc3.PNG)

**Comment by rchikhi** [comment]
Hi, to be honest I'm not sure if I have time to support this (old) code for windows. Did you try https://docs.microsoft.com/en-us/windows/wsl/install-win10 ?

**Comment by mhassankhan90** [comment]
Yes, I already have compiled it on Linux. Just trying to run it on Windows with MinGW compiler.
